packages/polling-controller/CHANGELOG.md

```diff
@@ -6,4 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/MetaMask/core/
+## [0.1.0]
+### Changed
+- Bump dependency on `@metamask/utils` to ^8.1.0 ([#1639](https://github.com/MetaMask/core/pull/1639))
```

Comment:
Hmm... it seems that 0.0.0 got released somehow. We should probably revoke that release.

In any case 0.1.0 seems like the proper initial release, yeah? Maybe we should denote it as such.

``` suggestion
### Added
- Initial release
```